src/routes/index.js

```diff
@@ -1,7 +1,14 @@
-import express from 'express';
+import { Router } from 'express';
 import api from './api';
 
-const router = express.Router();
+
+const router = new Router();
+router.get('/api/v1', (req, res) => {
+  res.status(200).json({
+    status: res.statusCode,
+    message: 'Welcome to Wolfsbane'
+  });
+});
```

Comment:
Remove this block of code. We already have a similar routing for the base route in the `index.js `sever entry file

Reply:
@uthdev remove the above block of code👆🏿